src/review.rs

```diff
@@ -87,7 +87,7 @@ fn prefix_lines(s: &str, prefix: &str) -> String {
 
     for line in s.lines() {
         if line.is_empty() {
-            ret += prefix;
+            ret += &(prefix.to_owned() + " \n");
```

Comment:
Nice find, this is an existing bug IIUC.

Nit: is the extra space necessary?

```suggestion
            ret += &(prefix.to_owned() + '\n');
```

Reply:
> Nit: is the extra space necessary?

IIRC I added this because `validate_review_file` expects a `< ` (note the space) to determine whether or not a line is quoted.

We check for both, `> `  (with a space) and `>` without a space in `parse_line`.  Maybe we should do the same in `validate_review_file`?